Author: Joone Hur <joone.hur@intel.com>

Do not check Gtk+ dependencies when use_ozone is defined.
    
All use of toolkit_use_gtk was removed, so we add use_ozone to skip Gtk+ 
dependencies: https://codereview.chromium.org/238633004

Upsteam patch: https://codereview.chromium.org/272743002/
diff --git a/build/linux/system.gyp b/build/linux/system.gyp
index dddf86a..76259a6 100644
--- src/build/linux/system.gyp
+++ src/build/linux/system.gyp
@@ -22,7 +22,7 @@
     'linux_link_libbrlapi%': 0,
   },
   'conditions': [
-    [ 'chromeos==0', {
+    [ 'chromeos==0 and use_ozone==0', {
       # Hide GTK and related dependencies for Chrome OS, so they won't get
       # added back to Chrome OS. Don't try to use GTK on Chrome OS.
       'targets': [
